cmd/beach/cmd/resource-download.go

```diff
@@ -59,6 +59,7 @@ Notes:
 func init() {
 	resourceDownloadCmd.Flags().StringVar(&instanceIdentifier, "instance", "", "instance identifier of the Beach instance to download from, eg. 'instance-123abc45-def6-7890-abcd-1234567890ab'")
 	resourceDownloadCmd.Flags().StringVar(&projectNamespace, "namespace", "", "The project namespace of the Beach instance to download from, eg. 'beach-project-123abc45-def6-7890-abcd-1234567890ab'")
+	resourceDownloadCmd.Flags().StringVar(&projectCluster, "cluster", "", "The cluster name of the Beach instance to download from, eg. 'h9acc4'")
```

Comment:
Just a nitpick: Internally in Beach we refer to a "cluster identifier" when it is something like `h9acc4` and "cluster name" when it is `flownative-beach-h9acc4`.